crates/astria-sequencer/src/state_ext.rs

```diff
@@ -243,3 +243,298 @@ pub(crate) trait StateWriteExt: StateWrite {
 }
 
 impl<T: StateWrite> StateWriteExt for T {}
+
+#[cfg(test)]
+mod test {
+    use cnidarium::StateDelta;
+    use tendermint::Time;
+
+    use super::{
+        StateReadExt as _,
+        StateWriteExt as _,
+    };
+
+    #[tokio::test]
+    async fn chain_id() {
+        let storage = cnidarium::TempStorage::new()
+            .await
+            .expect("failed to create temp storage backing chain state");
```

Comment:
High level observation on this usage of `expect` (applies to all instances below here as well):

1. the `expect` message should contain what you *expected* to happen, not the opposite, see this: https://doc.rust-lang.org/std/error/index.html#common-message-styles
2. I think in this case your intent is completely clear - you just want there to be a temp storage, so `unwrap()`ing is ok

Reply:
this is my bad as i put this in the other unit tests originally - @Lilyjjo feel free to replace all the `expect`s on these lines with `unwrap`